lib/ansible/modules/network/apconos/apconos_cert.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2018 Apcon.
+#
+# GNU General Public License v3.0+
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# Module to execute apconos Commands on Apcon Switches.
+# Apcon Networking
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: apconos_cert
+version_added: "2.9.0"
+author: "David Lee (@davidlee-ap)"
+short_description: install ssl ipv4 certificate on apcon network devices
+description:
+  - import and install ssl ipv4 certificate with specifying remote filename
+notes:
+  - tested against apcon iis+ii
+options:
+  command:
+    description:
+      - currently it is not being used in apconos_cert module.
```

Comment:
Why is it here then?

Reply:
Sanity test couldn't pass without it. It might be implemented in the future.

Reply:
If it's not used by the module, remove it from the module spec instead of adding it to the docs.

Reply:
I'll remove it.
Thanks!